contents/docs/feature-flags/common-questions.mdx

```diff
@@ -1,90 +1,178 @@
 
-[Local evaluation](/docs/feature-flags/bootstrapping-and-local-evaluation) enables you to pull flag definitions from the PostHog API, rather than making an individual `decide` call per flag. Using these definitions, you can resolve the flags of hundreds or thousands of users with a single API call. We therefore bill a single local evaluation call as being equivalent to 10 `decide`requests.
+### 2. Check you're calling `identify()` before fetching the flag
 
-Local evaluation payloads are typically used for much, _much_ more than 10 users, so local evaluation is often still more cost-effective than making individual `decide` requests. We also give all users their first 1M API calls (or 100k local evaluation calls) free each month, in addition to offering discounts for non-profit organizations and [startups](/startups).
+If you're [identifying]((/docs/product-analytics/identify)) users, they may see a different value for a feature flag before and after you call `identify()`. 
 
----
+Thus to ensure they experience consistent behavior, ensure you call `identify()` before fetching the feature flag.
 
-## How can I reduce the amount I pay for feature flags?
+### 3. An ad-blocker may be blocking calls
 
-Feature flags can get expensive if your code sends feature flag requests for each person.
+Check if an ad-blocker is interfering with PostHog calls. If this is the case, you can fix this by [deploying a reverse proxy](/docs/advanced/proxy).
 
-If this is happening on the client side, i.e. while using the `posthog-js` SDK, you have two options:
+### 4. User or group properties may not have been ingested yet
 
-1. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags_on_first_load` setting to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
+If your [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) depend on user or group properties, and you immediately fetch the feature flag after updating any of these properties, the properties may not be ingested in time to calculate the correct flag value.
 
-2. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags` setting to `true`. This stops automatically requesting feature flags. Instead, use the [bootstrap option](/docs/libraries/js#bootstrapping-flags) to load flags only once, via (server-side evaluation)[/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation], which makes no requests for flags from the client side.
+In this case, when making your `getFeatureFlag()` call, you can manually include in these properties in the method arguments.
```

Comment:
```suggestion
In this case, when making your `getFeatureFlag()` call, you can manually include in these properties as arguments.
```
Could we link to an example of this?

Reply:
Good idea. I'll add it in https://github.com/PostHog/posthog.com/pull/6561, since this is where I add that code